Gopkg.toml

```diff
@@ -35,44 +35,45 @@ required = [
 # Our master branch tracks knative/pkg master or a release.
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  #TODO(mattmoor): DO NOT SUBMIT
```

Comment:
😄

Reply:
This was me staging things to run CI and get feedback on changes.  In Serving we have the sockpuppet flag these: https://github.com/knative/serving/pull/5570#discussion_r325813525 (it is based on an internal presubmit check).

Once I get an `/lgtm` on serving, I'll start landing these.

Reply:
:+1: Can the sockpuppet do that for eventing too? :grin:

Reply:
It kinda looks like it should be, but I need to debug why it's not.